content/articles/2018-06-bench-1.0.1.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: bench 1.0.1
+date: '2018-06-28'
+slug: bench-1.0.1
+author: Jim Hester
+categories: [package]
+description: >
+  bench 1.0.1 is now available on CRAN. bench allows you to benchmark
+  code, tracking execution time, memory allocations and garbage
+  collections.
+photo:
+  url: https://unsplash.com/photos/nIkuMWT4Imc
+  author: Rula Sibai
+---
+
+[bench] is now available on CRAN!
+
+The goal of [bench] is to benchmark code, tracking execution time,
```

Comment:
Maybe add a by in place of the comma after "code" (on first read I thought tracking execution time was the second item in a list of goals)